source/catalog.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Седона</title>
+  </head>
+  <body class="page__body">
+    <header class="main-header">
+      <nav class="main-nav">
+        <a class="main-header__logo">
+        <img src="/source/img/header-logo.svg" width="185" height="96" alt="Город Седона.">
```

Comment:
Так будет правильно, так как корневая папка проекта `source` и пути должны строиться от неё:
```suggestion
        <img src="img/header-logo.svg" width="185" height="96" alt="Город Седона.">
```